tests/OSPSuite.Core.IntegrationTests/OSPSuite.Core.IntegrationTests.csproj

```diff
@@ -49,7 +49,10 @@
   <ItemGroup>
     <ProjectReference Include="..\..\src\OSPSuite.Core\OSPSuite.Core.csproj" />
     <ProjectReference Include="..\..\src\OSPSuite.Infrastructure.Castle\OSPSuite.Infrastructure.Castle.csproj" />
+    <ProjectReference Include="..\..\src\OSPSuite.Infrastructure.Import\OSPSuite.Infrastructure.Import.csproj" />
     <ProjectReference Include="..\..\src\OSPSuite.Infrastructure\OSPSuite.Infrastructure.csproj" />
+    <ProjectReference Include="..\..\src\OSPSuite.Presentation\OSPSuite.Presentation.csproj" />
+    <ProjectReference Include="..\..\src\OSPSuite.UI\OSPSuite.UI.csproj" />
```

Comment:
why is the UI requiured?

Reply:
in order to register the ICsvSeparatorSelector implementation. Since we are testing end to end importing from files, the instantiation of the objects needs a ICsvSeparatorSelector registration.

Reply:
Why is the CsvSeparatorSelector in UI then? Integrationtests should never rely on UI

Reply:
Actually you are right, I think I have to fake this anyway. I will change this.